### PR TITLE
Python server [WIP]

### PR DESCRIPTION
**Pull request by abdulmi**
* Pushing f(x) values to redis on the stream f_x_stream
* Pushing round status to a redis Pub/Sub channel

### CODE REVIEW

**Comment by abdulmi** [comment]
**example of data pushed to f_x_stream:**
Screen Shot 2020-05-19 at 2 16 45 AM
**example of data pushed to round_status pub/sub channel**
Screen Shot 2020-05-19 at 2 17 16 AM

File: server/main.py [hunk 1]
```diff
@@ -1,18 +1,54 @@
 import redis
+import time, datetime
+import threading
+import random
+
+r = redis.Redis(host='redis_db', port=6379, db=0)
+
+def round_runner():
+    round_counter = 1
```
Comment (gotibhai) [inline server/main.py:9]: Potentially, want to store this in redis?

File: server/main.py [hunk 2]
```diff
@@ -1,18 +1,54 @@
...
+        print("starting round {} at time {}".format(round_counter, time_in_five_minutes), flush=True)
+        # plan to start round in 5 minutes
+        r.publish("round_status", "starting_at:{}".format(time_in_five_minutes_epoch))
+
+        # sleep until current time is time_in_five_minutes_in_epoch
+        while datetime.datetime.now() < time_in_five_minutes:
+            time.sleep(5)
+        round_end_time = time_in_five_minutes + datetime.timedelta(minutes=1)
+        round_end_time_epoch = round_end_time.timestamp()
+        r.publish("round_status", "ending_at:{}".format(round_end_time_epoch))
```
Comment (gotibhai) [inline server/main.py:23]: Switch round-status with round counter?

Comment (abdulmi) [reply server/main.py:23]: round_status is just the name of the channel, doesn't make sense to name it round_counter no?

Comment (gotibhai) [reply server/main.py:23]: Oh my b, I thought it's k-v! Yeah makes sense.

File: server/main.py (hunk 1 above)
Comment (abdulmi) [reply server/main.py:9]: yea sure; could do

**Pull request merged**